qiskit_experiments/calibration_management/basis_gate_library.py

```diff
@@ -427,4 +432,32 @@ def _build_schedules(self, basis_gates: Set[str]) -> Dict[str, ScheduleBlock]:
 
             schedules["ecr"] = ecr
 
+        # RZXGate built from Echoed Cross-Resonance gate
+        if "rzx" in basis_gates:
+            with pulse.build(name="rzx") as rzx:
+                with pulse.align_sequential():
+                    pulse.reference("cr45p", "q0", "q1")
+                    pulse.reference("x", "q0")
+                    pulse.reference("cr45m", "q0", "q1")
+                    pulse.reference("x", "q0")
+
+            schedules["rzx"] = rzx
+
+        # Controlled NOT gate built from Echoed Cross-Resonance gate
+        if "cx" in basis_gates:
+            with pulse.build(name="cx") as cnot:
+                # Rz(np.pi / 2) on the control followed by ``x``.
+                pulse.shift_phase(np.pi / 2, pulse.DriveChannel(c_chan_idx))
```

Comment:
This doesn't really work. You are missing frame changes on associated control channels (and we cannot dynamically add instruction with parameterization framework) and this could break frame at circuit level. What you need to is basically consume device coupling graph and add mirror the frame change to all control channels. For example, when you change the phase of drive channel 0, and your qubit 0 is a target qubit of qubit 1 in the context of CR, then, you may need to propagate this phase to control channel 0 as well. The number of associated channels depends on the degree of the node in the coupling graph. Without phase propagation, following CR pulses for this qubit pair is driven at wrong phase.

This is why I'm refactoring the Qiskit Pulse now. We need some mechanism to define `logical port`, or `signal(envelope, frame)`. These representations allow us to define channels in term of qubits, namely,

```python
pulse.shift_phase(np.pi/2, DrivePort(qubit0))
```
or
```python
pulse.shift_phase(np.pi/2, Frame(qubit0))
```